src/thin_client.rs

```diff
@@ -12,6 +12,9 @@ use std::io;
 use std::net::{SocketAddr, UdpSocket};
 use transaction::Transaction;
 
+use influx_db_client as influxdb;
```

Comment:
I don't like how the `influx_db_client` crate needs to be pulled into for the `Point`/`Value` types, but I also don't want to wrap them with my own types in `metrics`.   Is there a better way?